ConsoleApp2/ConsoleApp2/ThreadsExample.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace ConsoleApp2
+{
+    class ThreadsExample
+    {
+        public void Example()
+        {
+            ThreadPool.QueueUserWorkItem(SideFunc);
+            ThreadPool.QueueUserWorkItem(More);
+            Console.WriteLine("Main func do something, then sleep");
+            Thread.Sleep(100);
+            Console.WriteLine("Main thread exits");
+
+            var task = Task.Run(() => Console.WriteLine("Hello Task!"));
+
+            var thread = new Thread(new ThreadStart(getMyName));
+            thread.Start();
+            Console.ReadKey();
+        }
+
+        private void getMyName()
```

Comment:
method name should start from capital letter

Reply:
Done